src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Assertions/DataLockRules/DataLockPeriodsRule.cs

```diff
@@ -18,10 +21,25 @@ public override void AssertDataLockEvents(DataLockContext context, LearnerResult
                     throw new Exception($"Event for price episode {expected.PriceEpisodeIdentifier} does not contain period {expected.Period}");
                 }
 
-                var actual = actualPeriods.FirstOrDefault(e => e.TransactionType == (int)expected.TransactionType);
+                if (expected.TransactionType == ReferenceDataModels.TransactionType.First16To18EmployerIncentive
```

Comment:
ReferenceDataModels namespace isn't required and can be removed on all the enums